packages/infra/lib/api-stack/api-service.ts

```diff
@@ -236,6 +247,40 @@ export function createAPIService({
     ),
   });
 
+  const nlb = new NetworkLoadBalancer(stack, `ApiNetworkLoadBalancer`, {
+    vpc,
+    internetFacing: false,
+  });
+  const nlbListener = nlb.addListener(`ApiNetworkLoadBalancerListener`, {
+    port: listenerPort,
+    protocol: Protocol.TCP,
+  });
+  const nlbTargetGroup = new NetworkTargetGroup(stack, `ApiNetworkTargetGroup`, {
+    port: listenerPort,
+    protocol: Protocol.TCP,
```

Comment:
alb using http and nlb using TCP. Will this be an issue?

Reply:
It should not be an issue, as its supported by AWS but this fails when we set HTTP - NLB accepts TCP, TLS, UDP, or TCP_UDP.